packages/kit/src/core/adapt/test/index.js

```diff
@@ -130,7 +130,19 @@ suite('prerender', async () => {
 		dest
 	});
 
-	assert.equal(glob('**', { cwd: prerendered_files }), glob('**', { cwd: dest }));
+	const expectedFiles = glob('**', { cwd: prerendered_files });
+	const actualFiles = glob('**', { cwd: dest });
+
+	// test if all files are present
+	assert.equal(expectedFiles, actualFiles);
```

Comment:
house style — we use snake_case instead of camelCase for anything that isn't public API, i.e. `expected_files` rather than `expectedFiles` (though single-word variables are better still). `assert.equal` takes `(actual, expected)`, not `(expected, actual)`

```suggestion
	const expected = glob('**', { cwd: prerendered_files });
	const files = glob('**', { cwd: dest });

	// test if all files are present
	assert.equal(files, expected);
```

Reply:
(I realise there is some adjacent camelCased code that snuck in in a previous PR; will address that separately)